### PR TITLE
added div to calc.py

### PR DESCRIPTION
**Pull request by jenniferdai1**
added div(a,b) function to calc.py to return the python division a/b

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -1,2 +1,6 @@
 def add(a, b):
     return a + b
+
+#returns a/b
```
Comment (stefanv) [inline calc.py]: Remove this left-over comment

File: calc.py [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+def div(a,b):
+	return a/b
```
Comment (18epedersen) [inline calc.py]: Make sure b is not zero

Comment (Seth-Park) [inline calc.py]: indentation

**Comment by jenniferdai1** [comment]
removed comment, checked for zero division, corrected indent

**Comment by 18epedersen** [comment]
:+1:

File: calc.py [hunk 3]
```diff
@@ -1,2 +1,7 @@
 def add(a, b):
     return a + b
+
+def div(a,b):
+    if b == 0:
```
Comment (stefanv) [inline calc.py]: How about:

```
try:
    return a / b
except ZeroDivisionError:
    raise ValueError("You should not try to divide by zero!")
```

**Comment by jenniferdai1** [comment]
yes that's better. fixed, review please

**Pull request merged**